File: troco.c
#include <stdio.h>
double preco, dinheiro, troco;
    int qtd;
int main()
{
    printf("Preco unitario do produto: ");
    scanf("%lf", &preco);
    printf("Quantidade comprada: ");
    scanf("%d", &qtd);
    printf("Dinheiro recebido: ");
    scanf("%lf", &dinheiro);
    troco=dinheiro-(preco*qtd);
    printf("Troco = %.2lf\n", troco);
	return 0;
}

File: L1, ling. C#/troco.cs
using System;
using System.Globalization;
class Program {
    static void Main() {
        int quantidade;
        double recebido, troco, total, preco;
        CultureInfo CI = CultureInfo.InvariantCulture;
        Console.Write("Preço unitário do produto: ");
        preco = double.Parse(Console.ReadLine(), CI);
        Console.Write("Quantidade comprada: ");
        quantidade = int.Parse(Console.ReadLine());
        Console.Write("Dinheiro recebido: ");
        recebido = double.Parse(Console.ReadLine(), CI);
        total = preco * quantidade;
        troco = recebido - total;
        Console.WriteLine($"Troco = R$ {troco:F2}");
    }
}
